docker-compose.yaml

```diff
@@ -12,9 +12,20 @@ services:
     ports:
       - "4306:3306"
     volumes:
-       - ./.data/mysql:/var/lib/mysql
+      - clrt-mysql-data:/var/lib/mysql
```

Comment:
If I delete all the docker volumes then the local DB mapping (.data) is lost. I need to reconfigure it again. what is the idea behind removing feature?

Reply:
I usually don't delete all docker volumes. Do you do this through like the Docker reset? The idea here was rather than having a directory on the filesystem we could just have it a a volume.

If you generally do delete all volumes I can revert this and just have the redis as a volume and this on the filesystem.

Reply:
I reverted this change.

Reply:
Thanks for reverting it. I delete it sometime I feel it make think better escaping from caching issue. But I could be very well be me doing this not everybody.

I want to follow best practice as others. But we can keep it for now